core-ui/src/components/HelmReleases/HelmReleasesDetails.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import LuigiClient from '@luigi-project/client';
+import { useTranslation } from 'react-i18next';
+import {
+  PageHeader,
+  useMicrofrontendContext,
+  useGetList,
+  Spinner,
+  prettifyNameSingular,
+  ResourceNotFound,
+} from 'react-shared';
+import { HelmReleaseData } from './HelmReleaseData';
+import { Link } from 'fundamental-react';
+import { HelmReleaseStatus } from './HelmReleaseStatus';
+import { OtherReleases } from './OtherReleases';
+
+export function HelmReleasesDetails({ releaseName }) {
+  const { t, i18n } = useTranslation();
+  const { namespaceId: namespace } = useMicrofrontendContext();
+  const breadcrumbItems = [
+    { name: t('helm-releases.title'), path: '/' },
+    { name: '' },
+  ];
+
+  const { data, loading } = useGetList(s => s.type === 'helm.sh/release.v1')(
+    `/api/v1/namespaces/${namespace}/secrets`,
+  );
+
+  if (loading) return <Spinner />;
+  const releaseSecret = data?.find(
+    r =>
+      r.metadata.labels.status !== 'superseded' &&
+      r.metadata.labels.name === releaseName,
```

Comment:
do we need this check, or maybe we could use the search query for the specific label in the initial call?